app/colorCoder.js

```diff
@@ -1,99 +1,44 @@
-			console.assert(testPair1.minorColor == "BROWN");
-
-			pairNumber = 5;
-            testPair1 = getColorFromPairNumber(pairNumber);
-            console.log(`[In]Pair Number: ${pairNumber},[Out] Colors:${testPair1}`);
-			console.assert(testPair1.majorColor == "WHITE");
-			console.assert(testPair1.minorColor == "SLATEGRAY");
-			 
-			pairNumber = 23;
-            testPair1 = getColorFromPairNumber(pairNumber);
-            console.log(`[In]Pair Number: ${pairNumber},[Out] Colors:${testPair1}`);
-			console.assert(testPair1.majorColor == "RED");
-			console.assert(testPair1.minorColor == "GREEN");
-			 
-            let testPair2 = new ColorPair();
-			testPair2.majorColor="YELLOW";
-			testPair2. minorColor ="GREEN";
-            pairNumber =getPairNumberFromColor(testPair2);
-            console.log(`[In]Colors: ${testPair2}, [Out] PairNumber: ${pairNumber}`);
-			console.assert(pairNumber==18);
-
-			testPair2 = new ColorPair();
-			testPair2.majorColor="RED";
-			testPair2. minorColor ="BLUE";
-            pairNumber =getPairNumberFromColor(testPair2);
-            console.log(`[In]Colors: ${testPair2}, [Out] PairNumber: ${pairNumber}`);
-			console.assert(pairNumber==6);
+const GetColorPair = require("./colorPair");
+const ProcessPairNumberFromColor = require("./getPairNumberFromColor.js");
+const FindColorsFromPairNumber = require("./getColorFromPairNumber.js");
+const getColor = new FindColorsFromPairNumber();
+const getPairNumber = new ProcessPairNumberFromColor();
+
+function test() {
+  pairNumber = 4;
+  let testPair1 = getColor.getColorFromPairNumber(pairNumber);
+  console.log(`[In]Pair Number: ${pairNumber},[Out] Colors:${testPair1}`);
+  console.assert(testPair1.majorColor == "WHITE");
+  console.assert(testPair1.minorColor == "BROWN");
+
+  pairNumber = 5;
+  testPair1 = getColor.getColorFromPairNumber(pairNumber);
+  console.log(`[In]Pair Number: ${pairNumber},[Out] Colors:${testPair1}`);
+  console.assert(testPair1.majorColor == "WHITE");
+  console.assert(testPair1.minorColor == "SLATEGRAY");
+
+  pairNumber = 23;
+  testPair1 = getColor.getColorFromPairNumber(pairNumber);
+  console.log(`[In]Pair Number: ${pairNumber},[Out] Colors:${testPair1}`);
+  console.assert(testPair1.majorColor == "RED");
+  console.assert(testPair1.minorColor == "GREEN");
+
+  let testPair2 = new GetColorPair();
+  testPair2.majorColor = "YELLOW";
+  testPair2.minorColor = "GREEN";
+  pairNumber = getPairNumber.getPairNumberFromColor(testPair2);
+  console.log(`[In]Colors: ${testPair2}, [Out] PairNumber: ${pairNumber}`);
+  console.assert(pairNumber == 18);
+
+  testPair2 = new GetColorPair();
+  testPair2.majorColor = "RED";
+  testPair2.minorColor = "BLUE";
+  pairNumber = getPairNumber.getPairNumberFromColor(testPair2);
+  console.log(`[In]Colors: ${testPair2}, [Out] PairNumber: ${pairNumber}`);
+  console.assert(pairNumber == 6);
+
+  console.log("------------------Color Manual --------------------------");
+  getColor.printColorsManual();
```

Comment:
`printColorsManual` needs to print a reference manual. Try asserting functionality that you would expect in a reference manual.